tests/Organization.py

```diff
@@ -185,6 +185,15 @@ def testGetHook(self):
     def testGetHooks(self):
         self.assertListKeyEqual(self.org.get_hooks(), lambda h: h.id, [257993])
 
+    def testGetHookDelivery(self):
```

Comment:
Can we please test all attributes of HookDelivery here? Same in `Repository.py`.